src/sql/magic.py

```diff
@@ -360,7 +360,7 @@ def interactive_execute_wrapper(**kwargs):
         with_ = self._store.infer_dependencies(command.sql_original, args.save)
         if with_:
             command.set_sql_with(with_)
-            print(f"Generating CTE with stored snippets : {', '.join(with_)}")
+            display.message("Generating CTE using display.py module")
```

Comment:
we should use the display.py module but the message should stay the same

Reply:
Got it. Update it now.